src/sentry/static/sentry/app/views/onboarding/project/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PlatformPicker from './platformpicker';
+import PlatformiconTile from './platformiconTile';
+import {t} from '../../../locale';
+
+const Project = React.createClass({
+  propTypes: {
+    next: React.PropTypes.func,
+    setPlatform: React.PropTypes.func,
+    platform: React.PropTypes.string,
+    setName: React.PropTypes.func,
+    name: React.PropTypes.string
+  },
+
+  submit() {
+    this.props.next();
+  },
+
+  render() {
+    return (
+      <div className="onboarding-info">
+        <h2>{t('Choose a language or framework')}</h2>
+        <PlatformPicker {...this.props} />
```

Comment:
no but it's a subset so what's the harm? can change this to 
`<PlatformPicker setPlatform={this.props.setPlatform} platform={this.props.platform}/>` if you want